### PR TITLE
fix: upgrade multus to an internal version v3.6.2-rc.1

### PR DESCRIPTION
**Pull request by hanxueluo**
**What type of PR is this?**

/area network

**What this PR does / why we need it**:

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @angao @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note
 upgrade multus to a internal version v3.6.2-rc.1
```

### CODE REVIEW

**Comment by hanxueluo** [comment]
multus 升级的前因后果：
https://github.com/caicloud/multus-cni/pull/20#issuecomment-752021848

File: inventory/deploy-cluster/group_vars/all/download.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ calico_policy_image_tag: "{{ calico_policy_version }}"
 pod_infra_image_repo: "{{ kube_image_repo }}/pause"
 pod_infra_image_tag: "{{ pod_infra_version }}"
 
-multus_image_repo: "{{ docker_image_repo }}/nfvpe-multus"
+multus_image_repo: "{{ docker_image_repo }}/multus"
```
Comment (WangMinjie114) [inline inventory/deploy-cluster/group_vars/all/download.yml]: 感觉这个 multus 已经成为了自研镜像，将 docker_image_repo 修改成 release_image_repo 吧 @hanxueluo

Comment (hanxueluo) [reply inventory/deploy-cluster/group_vars/all/download.yml]: 我是从下文 bridge-vlan 抄过来的，那下文也需要改成 release_image_repo ？

**Comment by hanxueluo** [comment]
/hold
wait for #78

**Pull request merged**